packages/jobs/lib/execution/sync.integration.test.ts

```diff
@@ -326,22 +321,22 @@ async function populateRecords(
 }
 
 async function seeds(records: UnencryptedRecordData[], trackDeletes: boolean) {
-    const envName = Math.random().toString(36).substring(7);
-    const env = await seeders.createEnvironmentSeed(0, envName);
-    const providerConfigKey = Math.random().toString(36).substring(7);
-    const model = Math.random().toString(36).substring(7);
+    const { env } = await seeders.seedAccountEnvAndUser();
+    const model = 'GithubIssue';
```

Comment:
Changed this test to be non-random (except the account id), bc there was an issue after I changed the seeders